util/reconciler/defaults.go

```diff
@@ -0,0 +1,21 @@
+package reconciler
+
+import (
+	"time"
+)
+
+const (
+	// DefaultLoopTimeout is the default timeout for a reconcile loop
+	DefaultLoopTimeout = 10 * time.Minute
```

Comment:
this might be a little bit optimistic if a single loop creates all the Azure infra / deletes all the azure infra for AzureCluster and AzureMachine.

Reply:
Perhaps, you're right. That makes me a little sad 😞. What do you think it should be?

Reply:
ARM template timeout is 90 minutes, perhaps that's a good baseline for now? 

PS: Makes me sad too, let's fix it!